connectivity/connectivity-demos/processes/rest/callOwnPersonRestService.p.json

```diff
@@ -131,7 +135,8 @@
         "startCustomFields" : [
           { "name" : "cssIcon", "value" : "fa fa-rss" }
         ],
-        "startName" : "1.3 Print REST service options"
+        "startName" : "1.3 Print REST service options",
+        "case" : { }

```

Comment:
looks like a bug I should address: with which designer did you manage to get this into json?

Reply:
I used 10.0.21